ui/app/components/totp/key-create.hbs

```diff
@@ -3,39 +3,30 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<form {{on "submit" @create}}>
+<form {{on "submit" (perform @onSubmit)}}>
   <div class="box is-sideless is-fullwidth is-marginless">
     <MessageError @model={{@model}} />
-    <NamespaceReminder @mode={{@mode}} @noun="TOTP key" />
```

Comment:
we do still need this, it shows when you're in a child namespace (though we've been historically pretty inconsistent with this) but we can hardcode `@mode="create"` instead of passing the arg since this will only ever show creation